src/full/Agda/Mimer/Mimer.hs

```diff
@@ -5,7 +5,7 @@ module Agda.Mimer.Mimer
   where
 
 import Control.DeepSeq (force, NFData(..))
-import Control.Monad ((>=>), (=<<), unless, foldM, when, zipWithM, filterM)
+import Control.Monad
```

Comment:
And here it goes, the qualified import discipline I praised in my initial review of Mimer.

Well yeah, falling back to our default style.

I have seen that HLS in VSCode can reconstruct the explicit import list, but I have not used it in large-scale projects like Agda.

Reply:
I don't mind explicit import lists for our own modules, especially when there are only a handful of things used, but for standard modules like `Control.Monad` I don't think it makes sense. What's the argument for explicit imports from `Control.Monad`, but not explicit imports from `Prelude`?

Reply:
Well, none really.

If we wanted PVP-style minor bump compatibility we would need to have explicit imports for everything everywhere.
We could do this through a `Agda.Prelude` module but I am a bit afraid this could become a bottleneck.  Not sure.  Maybe such a module would already be quite saturated so that we rarely have to add new imports from our dependencies there.  Then, it wouldn't be a bottleneck.  However, I don't know of any tooling that could create such a module for us.